### ISSUE TITLE
Compatibility with react 0.13.1

### ISSUE THREAD

**Issue by osasseville** [securingsincity/react-ace#14]
Hi, 
I'm getting this error when I use react 0.13.1 with webpack. I do not get this bug when I use react 0.12.0. My project is basically empty, it's only a tentative to mount a AceEditor.

```javascript
var React = require('react');
var Ace = require('react-ace');
React.render(
    <Ace
        mode="java"
        theme="github"
        name="uniq-ace"
    />,
    document.getElementById('ace-container')
);
```

![screen shot 2015-04-12 at 4 19 38 pm](https://cloud.githubusercontent.com/assets/3701019/7107313/c1dbe580-e12f-11e4-804f-396e88b8d64d.png)

**Comment by arqex** [comment]
Same error here running react 0.13.1 and react-ace 1.1.5

**Comment by securingsincity** [comment]
Should be fixed in 1.1.6. Please try to see if its fixed. @arqex @osasseville 

Thanks to @schloerke
